TI_mergesort.c

```diff
@@ -0,0 +1,72 @@
+#include <stdio.h>
+
+void merge(int arr[], int start, int mid, int end) {

```

Comment:
Das ist die Funktion zum Mergen, in meiner Notiz die ich euch in Discord geschickt habe ist es etwas ersichtlicher was genau das mergen macht.